ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -148,6 +148,11 @@ class AlertsView extends React.Component {
     const frameworkOptions = cloneDeep(frameworks);
     const ignoreFrameworks = { id: -1, name: 'all frameworks' };
     frameworkOptions.unshift(ignoreFrameworks);
+    const ignoreNonSheriffedFrameworks = {
```

Comment:
A better naming might be 
```suggestion
    const allSheriffedFrameworks = {
``` which aligns better with our approach to filter by sheriffed frameworks instead of excluding the non-sheriffed ones